client/src/screens/BallotListScreen.tsx

```diff
@@ -38,6 +38,10 @@ const BallotListScreen = () => {
     <NavigationScreen>
       <Header>
         <Prose maxWidth={false}>
+          <p>
+            <b>{printedBallots.length} official ballots</b> have been printed.
+            Election configured at <b>{configuredAt}</b>.
```

Comment:
```suggestion
            <strong>{printedBallots.length} official ballots</strong> have been printed.
            Election configured at <strong>{configuredAt}</strong>.
```

Reply:
yeah to be clear this is something shitty that I expect you will be replacing :)

Reply:
Gotcha. 😝